kiosk/src/index.tsx

```diff
@@ -18,19 +22,43 @@ function enableAnalytics() {
         stats["screen.availheight"] = screen.availHeight;
         stats["screen.innerWidth"] = window.innerWidth;
         stats["screen.innerHeight"] = window.innerHeight;
-        stats["screen.devicepixelratio"] = devicePixelRatio();
+        stats["screen.devicepixelratio"] = pxt.BrowserUtils.devicePixelRatio();
         const body = document.firstElementChild; // body
         if (body) {
             stats["screen.clientWidth"] = body.clientWidth;
             stats["screen.clientHeight"] = body.clientHeight;
         }
     }
-    tickEvent("kiosk.loaded", stats);
+    pxt.tickEvent("kiosk.loaded", stats);
 }
 
 window.addEventListener("DOMContentLoaded", () => {
+    const bundle = (window as any).pxtTargetBundle as pxt.TargetBundle;
+
+    pxt.options.debug = /dbg=1/i.test(window.location.href);
+    if (pxt.options.debug) pxt.debug = console.debug;
+
+    pxt.setupWebConfig((window as any).pxtConfig || pxt.webConfig);
+    pxt.setAppTarget(bundle);
+    // todo: handle this better?
+    if (pxt.BrowserUtils.isLocalHostDev()) {
+        // patch webconfig to refer to pxt serve instead of multiplayer serve
+        const wc = pxt.webConfig as any;
+
+        for (const key of Object.keys(wc)) {
```

Comment:
Now that I'm thinking of this we might be able to clear this out now with your localhost serving changes (or handle there)? have to test / see

Reply:
Oh you're probably right. Let me try that.